01who_we_are/02values.md

```diff
@@ -7,31 +7,73 @@ We take our core values very seriously. They're the underpinning for what we do,
 
 **Do great work**
 
-We value craftsmanship and set high standards for our work. We're proud of the clear, well-crafted products we create, as well as our professional demeanor.
+We value craftsmanship and set high standards for our work. We encourage each other to strive for those standards, helping each person grow, and make sure that we deliver a quality product for our customers and the larger open source community we are a part of.
+
+Some examples of this value in action are:
+
+- Our regular process of peer review for our code and content, where we provide feedback and encourage for improvement.
+- Contributing our standard high quality videos for free to enhance the Drupal community documentation.
 
 **Feed creativity**
 
-Creativity creates space for innovative ideas and inventive solutions. We embrace the full creative process, which includes failing, learning, and trying again.
+Creativity creates space for innovative ideas and inventive solutions. We embrace the full creative process, which includes failing, learning, and trying again, in a safe and respectful environment. We are open to creative change and are not afraid to change existing things as well as creating something new. Through openly sharing our new ideas in the wider world we aim to also foster creativity in others.
+
+Some examples of this value in action are:
+
+- We explore new ideas for how to deliver our content, from written tutorials to new hands-on project guides. Ideas start from customer feedback and result in features like the ability to access a pre-made site sandbox to avoid customers needing to set up a new site.
+- We re-evaluated our business strategy and decided to change the company name to better reflect our vision. 
+
 
 **Spread happiness**
 
-We believe in the power of happiness. We encourage playfulness, positivity, and kindness in everything we do.
+We believe in the power of happiness. We encourage playfulness, positivity, and kindness in everything we do. We try not to take ourselves too seriously and recognize that being approachable and kind makes learning more fun and less stressful. We are  aware of the impact our attitude has on the people around us and that impact spreads far beyond our personal and professional interactions.
+
+Some examples of this value in action are:
+
+- Creating fun examples in our learning materials, like the ice cream shop example for learning plugins or the batman cartoon for events.
+- Fun stickers! Our sparkly Drupal ponies continue to be a favorite even today.
```

Comment:
line 34: These two sentences could be merged into just: Sparkle Drupal ponies!

Reply:
Most people will have no idea what that means. This is aimed at people who don't know us well.